docs/gpu.md

```diff
@@ -74,3 +74,37 @@ Epoch 1 train begin 06:12:38
 ```
 ## AMP (AUTOMATIC MIXED PRECISION)
 AMP is very useful on GPU training and PyTorch/XLA reuse Cuda's AMP rule. You can checkout our [mnist example](https://github.com/pytorch/xla/blob/master/test/test_train_mp_mnist_amp.py) and [imagenet example](https://github.com/pytorch/xla/blob/master/test/test_train_mp_imagenet_amp.py). Note that we also used a modified version of [optimizers](https://github.com/pytorch/xla/tree/master/torch_xla/amp/syncfree) to avoid the additional sync between device and host.
+
+## Develop PyTorch/XLA on a GPU instance (build PyTorch/XLA from source with GPU support)
+
+1. Inside a GPU VM, create a docker container from a development docker image. For example:
+
+```
+sudo docker pull us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/development:3.8_cuda_11.8
+sudo apt-get install -y apt-transport-https ca-certificates curl gnupg-agent    software-properties-common
+distribution=$(. /etc/os-release;echo $ID$VERSION_ID)
+curl -s -L https://nvidia.github.io/nvidia-docker/gpgkey | sudo apt-key add -
+curl -s -L https://nvidia.github.io/nvidia-docker/$distribution/nvidia-docker.list | sudo tee /etc/apt/sources.list.d/nvidia-docker.list
+sudo apt-get update && sudo apt-get install -y nvidia-container-toolkit
+sudo systemctl restart docker
+sudo docker run --gpus all -it -d us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/development:3.8_cuda_11.8
+sudo docker exec -it $(sudo docker ps | awk 'NR==2 { print $1 }') /bin/bash
+```
+
+2. Build PyTorch and PyTorch/XLA from source.
+
+```
+git clone https://github.com/pytorch/pytorch.git
+cd pytorch
+USE_CUDA=0 python setup.py install
+
+git clone https://github.com/pytorch/xla.git
+cd xla
+XLA_CUDA=1 python setup.py install
```

Comment:
Is this env variable also relevant when installing a nightly wheel? This might be worth adding too

e.g. `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/11.7/torch-nightly-cp38-cp38-linux_x86_64.whl`

Reply:
`XLA_CUDA=1` is a build flag, shouldn't matter when you already have the whl